src/components/atoms/tooltip/tooltip.md

```diff
@@ -0,0 +1,19 @@
+```js props
+<Tooltip {props} />
+```
+
+---
+
+#### Examples
+
+Tooltip top:
+
+```js
+<Tooltip top />
```

Comment:
Should we also add the thing tooltip is on? I'm not sure how will the components align in that case.

Reply:
Yes, maybe we could have:
`<Tooltip top><Icon ... /></Tooltip>

Reply:
Higher order component! I like that